src/components/PlaybackControls/index.tsx

```diff
@@ -116,7 +123,50 @@ const PlayBackControls = ({
     const isStepForwardDisabled =
         compareTimes(time + timeStep, lastFrameTime, timeStep) === 1;
 
-    return (
+    const PlayPauseButton = (
+        <ViewportButton
+            tooltipText={isPlaying ? "Pause" : "Play"}
+            tooltipPlacement="top"
+            icon={isPlaying ? Pause : Play}
+            clickHandler={isPlaying ? pauseHandler : playHandler}
+            disabled={loading || isEmpty}
+            loading={loading}
+        />
+    );
+
+    const TimeSlider = (
+        <Slider
+            value={time}
+            onChange={handleTimeChange}
+            onAfterChange={handleSliderMouseUp}
+            tooltip={{ open: false }}
+            className={classNames(styles.slider, styles.item)}
+            step={timeStep}
+            min={firstFrameTime}
+            max={lastFrameTime}
+            disabled={loading || isEmpty}
+            range={false}
+        />
+    );
+
+    const minimalEmbeddedControls = (
+        <div className={styles.embedContainer}>
```

Comment:
nit: className could be `minimalControlsContainer` since this isn't going to be only embedded screens